styles/style.css

```diff
@@ -0,0 +1,160 @@
+/* NAVIGATION BAR */
+nav ul {
+    display: grid;
+    grid-template-columns: 6fr 2fr 1fr;
+}
+
+nav li {
+    list-style-type: none;
+}
+
+/* TAGLINE SECTION */
+.intro {
+    height: 25vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    border-bottom: 1px solid grey;
+}
+
+.intro a:link, a:visited {
+    color: gray;
+    padding: 14px 25px;
+    text-align: center;
+    text-decoration: none;
+    border: 1px solid;
+    border-radius: 8px;
+    display: inline-block;
+}
+
+.intro sub {
+    margin-bottom: 5vh;
+}
+
+/* QUOTES */
+.testimonial {
+    margin-top: 10vh;
+    height: 25vh;
+    text-align: left;
+    border-bottom: 1px solid grey;
+}
+
+.testimonial p {
+    text-align: right;
+    margin-top: 10vh;
+    margin-right: 25vh;
+    
+}
+
+/* WHY SECTION */
+.selling-points {
+    padding-bottom: 5vw;
+    border-bottom: 1px solid grey;
+}
+
+.selling-points h2 {
+    text-align: center;
+    padding-top: 3vw;
+}
+
+.selling-points img {
+    float: left;
+    padding-right: 1.5vw;
+    padding-bottom: 1vw;
+}
+
+.selling-points ul {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    list-style-type: none;
+
+    justify-content: space-between;
+
+    padding: 0;
+}
+
+.selling-points ul li {
+
+    margin: 16px;
+} 
+
+.selling-points strong,
+.selling-points p {
+    max-width: 25vw;
+    justify-content: center;
+}
+
+/* MEET THE TEAM */
+
+.team {
+    background-color: lightcoral;
+    padding-bottom: 3vw;
+    padding-top: 3vw;
+}
+
+.team h2 {
+    text-align: center;
+    /* padding-top: 5vw; */
+    padding-bottom: 5vw;
+}
+
+.team ul {
+    display: flex;
```

Comment:
also great use of Flexbox. It really makes the responsiveness of your site work well.